packages/hardhat-core/package.json

```diff
@@ -23,7 +23,7 @@
     "hardhat": "internal/cli/cli.js"
   },
   "engines": {
-    "node": "^12.0.0 || ^14.0.0 || ^16.0.0"
+    "node": "^12.0.0 || ^14.0.0 || ^16.0.0 || ^17.0.0"
```

Comment:
I'm guessing this was just for your local development? Not needed as part of this PR, right?

Reply:
it is, but this file really should be updated for people with newer versions of NPM.

Reply:
Indeed, but as far as I know there is an outstanding issue or two with compatibility with 17, which is why we haven't made that update yet.